python-client/giskard/ml_worker/core/suite.py

```diff
@@ -9,6 +9,7 @@
 from giskard.core.model import Model
 from giskard.ml_worker.core.dataset import Dataset
 from giskard.ml_worker.testing.registry.registry import create_test_function_id
+from ml_worker_pb2 import SingleTestResult
```

Comment:
Could you please replace this line with `from giskard.ml_worker.generated.ml_worker_pb2 import SingleTestResult
SingleTestResult`